builtins/amp-img.js

```diff
@@ -82,9 +95,7 @@ export class AmpImg extends BaseElement {
     if (this.img_) {
       return;
     }
-    if (!this.srcset_) {
```

Comment:
I'm gonna try to speed-up `parseSrcset` using more `substring` and less regex. Hopefully that will give us a little speed bump to be able to put back inside `buildCallback`